freesound/static/bw-frontend/src/components/modal.js

```diff
@@ -85,19 +85,22 @@ const bindConfirmationModalElements = (container) => {
 
 // Logic to bind default modals
 
-const handleDefaultModal = (modalUrl, modalActivationParam, atPage) => {
-  if ((atPage !== undefined) && modalUrl.indexOf('&page') == -1){
-    modalUrl += '&page=' + atPage;
-  }
+const handleDefaultModal = (modalUrl, modalActivationParam) => {
   handleGenericModal(modalUrl, undefined, undefined, true, true, modalActivationParam);
 }
 
+const handleDefaultModalWithForm = (modalUrl, modalActivationParam) => {
+  handleGenericModalWithForm(modalUrl, undefined, undefined, (req) => {showToast('Form submitted succesfully!')}, undefined, true, true, modalActivationParam, true);
```

Comment:
You are enabling the reload on success by default, and also not providing a way to disable it unless `handleGenericModalWithForm` is used directly instead of an automatic binding. You should probably add a parameter to `handleDefaultModalWithForm` (e.g. `reloadOnSuccess`) which defaults to false, and then in the bind function, see if the element has a specific dataset property to decide if `reloadOnSuccess` should be true or false (e.g. `data-reload-on-success`)

Reply:
Ok, it might be a good idea to simply pass `element` (also when binding), you can do it like that